src/tools/simeng/main.cc

```diff
@@ -82,13 +84,25 @@ int main(int argc, char** argv) {
       coreInstance->getInstructionMemory();
 
   // Output general simulation details
-  std::cout << "[SimEng] Running in " << coreInstance->getSimulationModeString()
-            << " mode" << std::endl;
+  std::cout << "[SimEng] Running in "
+            << simeng::config::SimInfo::getSimModeStr() << " mode" << std::endl;
   std::cout << "[SimEng] Workload: " << executablePath;
   for (const auto& arg : executableArgs) std::cout << " " << arg;
   std::cout << std::endl;
-  std::cout << "[SimEng] Config file: " << configFilePath << std::endl;
-  std::cout << "[SimEng] ISA: " << coreInstance->getISAString() << std::endl;
+  std::cout << "[SimEng] Config file: "
+            << simeng::config::SimInfo::getConfigPath() << std::endl;
+  std::cout << "[SimEng] ISA: " << simeng::config::SimInfo::getISAString()
+            << std::endl;
+  std::cout << "[SimEng] Local Special File directory: ";
```

Comment:
Should the print out say "Auto-Generated Special File Directory" given the used files will always be local?

Reply:
Good point and I'm happy with the suggested change